eng/Versions.props

```diff
@@ -42,7 +39,7 @@
   </PropertyGroup>
   <PropertyGroup>
     <!-- Dependencies from https://github.com/nuget/nuget.client -->
-    <NuGetBuildTasksPackageVersion>5.5.0-preview.2.6355</NuGetBuildTasksPackageVersion>
```

Comment:
Was the problem before simply that we had multiple entries for `<NuGetBuildTasksPackageVersion>` in the Version.props file? DARC was then simply updating the first one but not the second hence we did not get any true dependency flow here?

If so can we please file a bug in arcade. This seems like a really easy case to catch and given it's likely to contribute to a servicing event it's one we should definitely consider implementing.